Dockerfile

```diff
@@ -67,3 +67,9 @@ RUN mkdir /opt/rwd/ && \
     | tar -xzC /opt/rwd --strip-components=1 && \
     cd /opt/rwd && \
     pip install --no-cache-dir -r requirements.txt
+
+RUN pip install flask==0.10.1
```

Comment:
Thinking about it more, I think we'll want to develop and test the API outside the context of the container, so including a separate requirements file in `src/api` may make sense.

Still curious for input from others, though.